.github/config/raspi-2.json

```diff
@@ -1,7 +1,7 @@
 {
   "docker_service": "build-raspi",
   "on_device_test": {
-    "enabled": false,
+    "enabled": true,
```

Comment:
Are we enabling odt on Raspi-2?  In general, I would like to enable odt on main.

Reply:
Right now this is just to test if this fixes any issues we were seeing